source/tutorials/building-smart-contract-only-dapps.md

```diff
@@ -0,0 +1,183 @@
+title: Building Smart Contract only DApps with Embark
+layout: tutorial
+---
+
+Building decentralized applications often involves many parts and components, such as Smart Contracts and a front-end, that have to play well together, in order to provide users the best experience possible. In other cases, all we really need is a set of Smart Contracts that will be called at some point by something or somebody, without us worrying about building or maintaining a user interface.
+
+Embark enables us to implement either of those scenarios and in this article we're going to explore how to build a decentralized applications where Smart Contracts are the primary focus.
```

Comment:
I think this is really a matter of defining "decentralized app". I personally would say that just Smart Contracts are also decentralized apps, they are the barebone of the application after all and fully decentralized by default. This can not be said for the front-end that might come with it (e.g. front-ends are often still hosted on centralized services and are therefore anything but decentralized).

Happy to reword this if there's general consensus that a DApp is only a DApp when it has a front-end.